core/vm/instructions.go

```diff
@@ -592,9 +593,11 @@ func opCreate(pc *uint64, evm *EVM, contract *Contract, memory *Memory, stack *S
 		stack.push(addr.Big())
 	}
 	contract.Gas += returnGas
-
 	evm.interpreter.intPool.put(value, offset, size)
 
+	if suberr == errExecutionReverted {
+		return res, nil
```

Comment:
If I follow along here.... 

* A `revert` during initcode, 
* Would place a `0` on the stack, 
* But make the `revert`:ed data available for `returndatacopy`

?